### DISCUSSION TITLE
Label articles: use labels as sole in- & exclusion criteria

### DISCUSSION THREAD

**Discussion by verweijs** [asreview/asreview#1042] [Q&A]
I am planning to use ASReview, but a crucial factor in my screening is labelling both in- and excluded articles for later research (as would be possible manually in [Rayyan](https://www.rayyan.ai/) for example). 

To describe the situation, let’s say I have to screen a total of 5000 articles for my systematic review. For the main analysis, I will feed 500 articles as input to ASReview. Of these 100 articles, 40 are `included` based on all the inclusion criteria, the other 460 are `excluded` based on all the exclusion criteria. From what I understand, ASReview will use this input to predict the status of the remaining 4500 articles. 

Now of these 5000 articles, I would also like to know which articles are safety studies and which articles are effectiveness studies, for possible later research. In _Rayyan_ I would just apply a label on each in- and excluded article stating `safety study` or `effectiveness study` (a combination of a effectiveness and safety study can be labelled as `safety study` in my case).

Now since it seems that labeling isn’t possible in ASReview I am thinking of a workaround: Run the ASReview model again for every label applied. In the case described above that would mean:
- **First**, from the 500 articles to feed the model as input, `include` _only_ the articles which are a safety study or combination (e.g. 340) and `exclude` the rest (e.g. 260). 
- **Second**, Since the exclusion set of the first set would also contain other studies besides effectiveness studies, one needs to run a new ASReview model; this time _only_ including articles which are sole effectiveness studies, excluding the rest.

**Q**: Do you know if ASReview will correctly predict the labelling (safety or effectiveness) for the remaining 4500 articles in the case described above, taken into account the chance of errors being similar to the errors made when screening the articles manually? Ofcourse, if there are _alternative ways_ of adding labels via ASReview I would like to know that as well!

Note that safety studies are quite varying as well (all adverse reactions etc.), so I am also wondering if the model could handle such a various included selection (especially since the included selection also contains combinations with effectiveness studies).

When using >2 labels for a single category (e.g. besides safety studies and effectiveness studies I am also interested in animal studies) one needs to run ASReview for every additional label: 
- First including only "safety studies", excluding the rest.
- Second time including only "effectiveness studies", excluding the rest.
- Third time including only "animal studies", excluding the rest

**Comment by Rensvandeschoot** [comment]
You already answered your question :-) 

I would indeed recommend using Active Learning (not just ASReview, but any software implementing AL) to screen records with just one research question in mind. If you mix multiple queries, you might end up at a plateau, and it will take many random iterations before the machine suggests a paper in another domain. Multinominal classification tasks are technically possible, but not implemented. 

Be aware that AL does not seek or highlight specific words. If you want to do this in ASReview, you can do so via a[ plug-in](https://github.com/asreview/asreview/discussions/995). 

In v1.0 you will be able to add [notes](https://github.com/asreview/asreview/pull/687), so you can add information on why you included or excluded a record (or any note).
